css/properties/-moz-user-modify.json

```diff
@@ -0,0 +1,72 @@
+{
+  "css": {
+    "properties": {
+      "-moz-user-modify": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Mozilla/CSS/-moz-user-modify",
+          "support": {
+            "chrome": {
+              "prefix": "-webkit-",
```

Comment:
I kind of thought the same thing when running through this file, but decided to name it `-moz-user-modify` as that's what the link refers to.  But since you're indicating that it breaks compatibility, creating a commit to accommodate now.  :wink: